tests/python/unittest/test_tir_lower_match_buffer.py

```diff
@@ -22,10 +22,10 @@
 
 
 def _check(original, transformed):
-    mod = tvm.IRModule.from_expr(original)
+    mod = tvm.IRModule.from_expr(original.with_attr("global_symbol", "main"))
```

Comment:
Should `from_expr` include a global symbol by default? The only reason I hesitated on implementing that is because Relay and Relax also use `from_expr` and I didn't want to mess with that.